integration_tests/test_builtin_type.py

```diff
@@ -0,0 +1,27 @@
+from lpython import i32, f64, Const
```

Comment:
If `Const` is unused, remove it.

Reply:
It was used before, but when we decided to make it work exactly like Python, we removed checks for `Const` types.